spec/features/book_index_redirect_spec.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.feature 'Book index redirect', type: :feature do
```

Comment:
Strange separation 🤔
It is enough to have just `book_index_spec.rb` where you will test books and their search